python/deltalake/table.py

```diff
@@ -1022,6 +1022,8 @@ def to_pyarrow_dataset(
         partitions: Optional[List[Tuple[str, str, Any]]] = None,
         filesystem: Optional[Union[str, pa_fs.FileSystem]] = None,
         parquet_read_options: Optional[ParquetReadOptions] = None,
+        schema: Optional[pyarrow.Schema] = None,
+        as_large_types: bool = False,
```

Comment:
Doc description is missing for this param. I would also mention if the schema is passed that takes precedence over as_large_types

Reply:
done!